package.json

```diff
@@ -23,15 +23,18 @@
 	"homepage": "https://github.com/juliuste/sainte-lague",
 	"repository": "juliuste/sainte-lague",
 	"bugs": "https://github.com/juliuste/sainte-lague/issues",
-	"main": "./index.js",
+	"main": "./dist/index.js",
 	"files": ["index.js"],
 	"dependencies": {
 	},
 	"devDependencies": {
+		"babel-cli": "^6.24.1",
+		"babel-preset-es2015": "^6.24.1",
+    "babelify": "^7.3.0"
 	},
 	"scripts": {
 		"test": "node test.js",
-		"prepublish": "npm test"
+		"prepublish": "npm test && babel index.js --presets babel-preset-es2015 --out-dir dist"
```

Comment:
This setup should also be run by Travis to make sure it works. For clarity, you could move it into an npm script named `transpile`.